include/seqan3/range/views/pairwise_combine.hpp

```diff
@@ -65,18 +64,12 @@ class pairwise_combine_view : public std::ranges::view_interface<pairwise_combin
     /*!\name Constructors, destructor and assignment
      * \{
      */
-    //!\brief Default Default-Constructor.
-    constexpr pairwise_combine_view() = default;
-    //!\brief Default Copy-Constructor.
-    constexpr pairwise_combine_view(pairwise_combine_view const &) = default;
-    //!\brief Default Move-Constructor.
-    constexpr pairwise_combine_view(pairwise_combine_view &&) = default;
-    //!\brief Default Copy-Assignment.
-    constexpr pairwise_combine_view & operator=(pairwise_combine_view const &) = default;
-    //!\brief Default Move-Assignment.
-    constexpr pairwise_combine_view & operator=(pairwise_combine_view &&) = default;
-    //!\brief Default Destructor.
-    ~pairwise_combine_view() = default;
+    pairwise_combine_view() = default; //!< Defaulted.
+    pairwise_combine_view(pairwise_combine_view const &) = default; //!< Defaulted.
+    pairwise_combine_view(pairwise_combine_view &&) = default; //!< Defaulted.
+    pairwise_combine_view & operator=(pairwise_combine_view const &) = default; //!< Defaulted.
+    pairwise_combine_view & operator=(pairwise_combine_view &&) = default; //!< Defaulted.
+    ~pairwise_combine_view() = default; //!< Defaulted.
```

Comment:
Why are these not constexpr anymore?

Reply:
If you `default` constructors they implicitly are `constexpr` and `noexcept`, if they can be `constexpr` and `noexcept`.

Svenja requested this change in https://github.com/seqan/seqan3/pull/1914#discussion_r441996820 (the PR which override my changes) for the view::iterator class, I made this consistent across the file.